src/lips.js

```diff
@@ -126,10 +126,10 @@ function log(x, regex = null) {
         }
         var value = global_env.get('repr')(x);
         if (regex === null || regex instanceof RegExp && regex.test(value)) {
-            console.log(global_env.get('type')(x) + ": " + value);
+            console.log(global_env.get('type')(x) + ": " + value); // @jcubic can you enter what this console.log returns here
```

Comment:
`log` is a debug function. `global_env.get('type')` returns the `type` Scheme function that is called with `x` which in turn returns the type of the variable. So it prints something like `string: "hello"` when called with `log("hello")` But it also works with LIPS types like LString. And `repr` will return the representation of the variable as Scheme sees it.

Reply:
should be deleted now